spec/models/public_body_spec.rb

```diff
@@ -2067,7 +2067,9 @@ def set_default_attributes(public_body)
     end
 
     it 'does not mark the authority for reindexing' do
-      pending "This should pass, but no_xapian_reindex is not preventing reindexing"
+      # Call public_body so that any unrelated indexing events are created
+      # before we call update_counter_cache
+      public_body
```

Comment:
To make this extra clear we could change this line to `public_body.save` which shouldn't hit the DB due to there being no changes

Reply:
Fixed in 533c34fac